twake/backend/node/src/core/platform/services/database/services/orm/connectors/cassandra/cassandra.ts

```diff
@@ -289,6 +290,31 @@ export class CassandraConnector extends AbstractConnector<
       result = false;
     }
 
+    // --- Create indexes --- //
+    if (entity.options.globalIndexes) {
+      for (const globalIndex of entity.options.globalIndexes) {
+        const indexName = globalIndex.join("_");
+        const indexDbName = "index_" + md5(indexName);
```

Comment:
template literal :p

Reply:
Rhoo, pasted from php I missed this one :(